src/home_robot/setup.py

```diff
@@ -17,6 +17,7 @@
     "sophuspy",
     "trimesh",
     "pin==2.6.17",
+    "pinocchio"
```

Comment:
@hello-fazil thanks for pushing the change! I will move this to `home_robot_hw` as this is not needed in simulation

Reply:
@hello-fazil seems like it is already present inside `src/home_robot_hw/environment.yml` as a dependency. This should get installed in step 1 right?

Any reason for not using the mamba environment on real robot?
> I think the [src/home_robot](https://github.com/facebookresearch/home-robot/tree/ovmm_new_setup_testing/src/home_robot) package depends on Pinnocio python package and it is not installed robot side without mamba environment

Reply:
Not any particular reason. But I was following the steps mentioned in [install_robot.md](https://github.com/facebookresearch/home-robot/blob/ovmm_new_setup_testing/docs/install_robot.md) which does not mention the environment install on the robot . And it also mentioned the workspace to be built to the native environment.

Reply:
I see. I am not sure if parts of this README are outdated. The participants are supposed to follow the [main README](https://github.com/facebookresearch/home-robot/tree/ovmm_new_setup_testing#1-create-your-environment). @cpaxton can you please confirm this and maybe also check if the [install_robot.md](https://github.com/facebookresearch/home-robot/blob/ovmm_new_setup_testing/docs/install_robot.md) is up-to-date?

Reply:
There is no environment install on the robot. Just supposed to do the requirements file.

Reply:
you do *not* use Conda on the robot

Reply:
@hello-fazil - pinocchio is actually the *wrong* package, I believe. And we should probably update install_robot.md to use this: https://github.com/facebookresearch/home-robot/blob/main/src/home_robot/requirements.txt

Reply:
"pin==2.6.17",
this is pinocchio, it should already be installed in setup.py

Reply:
Thank you for the clarification. I can revert my changes. 
This case was falling during my testing and got resolved only after installing pinacchio, could be some other issue.

Reply:
@hello-fazil  I will revert the last commit so that we can merge this PR. Let us know if the issue persists, we can resolve them in another PR.


> And we should probably update install_robot.md to use this: https://github.com/facebookresearch/home-robot/blob/main/src/home_robot/requirements.txt
@cpaxton I added a `pip install -r requirements.txt` in `install_robot.md`

Reply:
Oops missed your comment @hello-fazil. I already reverted it. Yes, please test again if you get a chance.